charts/forkmon/values.yaml

```diff
@@ -25,7 +25,7 @@ customCommand: [] # Only change this if you need to change the default command
 network: ""
 
 # -- Config file
-# @default -- See `values.yaml`
+# @default -- See `values.toml`
```

Comment:
This should be "values.yaml" . This will appear on the main Readme and suggests the user to check the "values.yaml" file to check for the default values for the config field.

Reply:
ah for some reason I thought it was referring to the values.toml that the config file is in. Ok, gonna revert this.